README.md

```diff
@@ -15,10 +15,14 @@ This module has been tested to work on the following systems with Puppet v3
 
  * EL 5
  * EL 6
+ * EL 7
  * Suse 11
+ * Suse 12
  * Solaris 10
  * Solaris 11
  * Ubuntu 12.04 LTS
+ * Ubuntu 14.04 LTS
```

Comment:
Thanks @philippvh!

The `metadata.json` should also be updated to reflect the supported platforms.

Reply:
Thanks for the hint, just updated that.